rpc_jobs/multi_node_aio.yml

```diff
@@ -174,6 +175,10 @@
           }}// public cloud node
         }} catch (e){{
             currentBuild.result = 'FAILURE'
+            if("{trigger}" == "post-merge"){{
+              github.create_issue(env.BUILD_TAG,
```

Comment:
The commit order seems a bit wonky, this commit is added before the one introducing the create_issue function.